src/code/entities/07_horse_piece.asm

```diff
@@ -183,12 +185,12 @@ jr_007_76C2:
     inc  a                                        ; $76E5: $3C
     ld   [hl], a                                  ; $76E6: $77
 
-func_007_76E7::
+.func_76E7
```

Comment:
Why the local label?

Reply:
Because it's only locally used, the code flow is a bit weird there, which is why I haven't fully grasped what it is doing.

But I mostly made this into a local label to make the .ret below it also as a local label. Which then makes that whole "function" read as one thing, which just that part at `.func_76E7` used twice.

Reply:
He, indeed, there's a few place in the code that use a ret label after an embedded function… Makes sense then :)